lib/internal/Magento/Framework/ObjectManager/InterceptableValidator.php

```diff
@@ -39,7 +52,7 @@ private function isInterceptable($instanceName)
     {
         return !is_subclass_of(
             $instanceName,
-            '\\' . \Magento\Framework\ObjectManager\Code\Generator\Proxy::NON_INTERCEPTABLE_INTERFACE
+            '\\' . \Magento\Framework\ObjectManager\NoninterceptableInterface::class
```

Comment:
Do you have any ideas why we should add the backslash prefix here?

Reply:
I preserved backslash prefix because it was already there and it worked, to minimize PR changes. However, it works both with and without backslash prefix:

```
interface AI {}
class A implements AI {}

$a = new A;

var_dump(is_subclass_of($a, 'AI'));
var_dump(is_subclass_of($a, '\AI'));
```

```
bool(true)
bool(true)
```

So we can either remove it or leave it as is.